src/events/mouse.js

```diff
@@ -929,4 +972,45 @@ p5.prototype._onwheel = function(e) {
   }
 };
 
+/**
+ * <p>The function <a href="#/p5/requestPointerLock">requestPointerLock()</a>
+ * locks the pointer to its current position and makes it invisible.
+ * Use <a href="#/p5/deltaX">deltaX</a> and <a href="#/p5/deltaY">deltaY</a> to get the difference the mouse was moved since
+ * the last call of draw</p>
+ * <p>Note that not all browsers support this feature</p>
+ * <p>This enables you to create experiences that aren't limited by the mouse moving out of the screen
+ * even if it is repeatedly moved into one direction. </p>
+ * <p>For example a first person perspective experience</p>
+ *
+ * @method requestPointerLock
+ * @example
+ * <div class="notest">
+ * <code>
+ * //click the canvas to lock the pointer
+ * //press ESC to exit lock mode
+ * function draw() {
+ *   background(237, 34, 93);
+ * }
+ * function mouseClicked() {
+ *   requestPointerLock();
+ * }
+ * </code>
+ * </div>
+ *
+ * @alt
+ * black 50x50 rect moves up and down with mouse move while no cursor is visible
+ *
+ */
+p5.prototype.requestPointerLock = function() {
+  // pointer lock object forking for cross browser
+  var canvas = this._curElement.elt;
+  canvas.requestPointerLock =
+    canvas.requestPointerLock || canvas.mozRequestPointerLock;
+  if (!canvas.requestPointerLock) {
+    console.log('requestPointerLock is not implemented in this browser');
+    return;
```

Comment:
What do you think about returning `false` here if its not supported? and return `true` at the end of the function if it is? That way developers can tell if its going to work or not and do something else if not? or maybe that's overkill.

Reply:
done

Reply:
thank you!